spec/factories/visits.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :visit, class: Visit do
+    user
+    created_at { rand(3.month.ago..Time.now) }
```

Comment:
attribute first
associations after

Reply:
actually it's ok :)  http://prntscr.com/i0apzu from official page

but use trait. 
I don't want to make this change global.